packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -537,14 +535,14 @@ class ScrollingFlameChartRowState<V> extends State<ScrollingFlameChartRow>
 }
 
 extension NodeListExtension on List<FlameChartNode> {
-  List<Rect> toPaddedZoomedRects({
+  List<SimpleInterval> toPaddedZoomedIntervals({
     @required double zoom,
     @required double chartStartInset,
     @required double chartWidth,
   }) {
-    return List<Rect>.generate(
+    return List<SimpleInterval>.generate(
```

Comment:
another name like Interval that isn't already used by Flutter is `Range`.
Either SimpleInterval or `Range` is fine with me.

Reply:
i like range. done.